rlberry/agents/agent.py

```diff
@@ -485,32 +490,74 @@ class AgentWithSimplePolicy(Agent):
 
     @abstractmethod
     def policy(self, observation):
-        """Returns an action, given an observation."""
+        """
+        Abstract method to be overwriten by the 'inherited agent' developer.
```

Comment:
Indeed, some parts were generated automatically, should be cleaner now

Reply:
If you talk about the "Abstract method to be overwriten by the 'inherited agent' developer." it's not ChatGPT, but me. 
It's because the decorator because the decorator didn't appear in the documentation. (and the function do nothing)

If you talk about the rest of the text I don't know.